WebApplication/Backend/Repositorys/FeedbackRepository.cs

```diff
@@ -1,24 +1,154 @@
+        ///</returns>
+        ///<exception>
+        ///if any exception occurs method will return null
+        ///</exception>
         internal List<Feedback> GetAllFeedbacks()
         {
-            throw new NotImplementedException();
+                string sql1 = "Select * from feedbacks";
+                MySqlCommand cmd1 = new MySqlCommand(sql1, connection);
+                MySqlDataReader rdr = cmd1.ExecuteReader();
+                List<Feedback> resultList = new List<Feedback>();
+                while (rdr.Read())
+                {
+                    Feedback entity = new Feedback();
+                    entity.SerialNumber = (string)rdr[4];
+                    entity.PatientId = (String)rdr[3];
+                    entity.Text = (String)rdr[0];
+                    entity.Date = Convert.ToDateTime(rdr[2]);
+                    entity.Approved = (Boolean)rdr[1];
+                    resultList.Add(entity);
+
+                }
+                connection.Close();
+                return resultList;
+        }
+
+        internal List<Feedback> GetFeedbacks(Boolean approvedF)
+        {
+                string sql1;
+                if(approvedF)
+                    sql1 = "Select * from feedbacks where approved is true";
+                else
+                    sql1 = "Select * from feedbacks where approved is false";
+
+                MySqlCommand cmd1 = new MySqlCommand(sql1, connection);
+                MySqlDataReader rdr = cmd1.ExecuteReader();
+                List<Feedback> resultList = new List<Feedback>();
+                while (rdr.Read())
+                {
+                    Feedback entity = new Feedback();
+                    entity.SerialNumber = (string)rdr[4];
+                    entity.PatientId = (String)rdr[3];
+                    entity.Text = (String)rdr[0];
+                    entity.Date = Convert.ToDateTime(rdr[2]);
+                    entity.Approved = (Boolean)rdr[1];
+                    resultList.Add(entity);
+
+                }
+                connection.Close();
+                return resultList;
+
         }
 
         internal Feedback GetFeedbackById(string feedbackId)
         {
             throw new NotImplementedException();
         }
-        internal void SetFeedbackApprovedValue(Feedback feedback)
+
+        ////Vucetic Marija RA157/2017
+        /// <summary>
+        ///set value of atrribute Approved
+        ///</summary>
+        ///<returns>
+        ///true if sucessful,else false
+        ///</returns>
+        ///<exception>
+        ///if any exception occurs method will return null
+        ///</exception>
+        ///<param name="feedback"> Feedback type object
+        ///</param>
+        internal void SetFeedbackApprovedValue(FeedbackDTO feedback)
         {
-            throw new NotImplementedException();
+                string[] dateString = feedback.Date.ToString().Split(" ");
+                string[] partsOfDate = dateString[0].Split("/");
```

Comment:
I don't think you need to parse date this way, there are getters for day, month, year, hour, minute...

Reply:
Actually I think there's a method in DateTime class that returns string in any format you need.
https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings
It's really well supported unlike Date in Java 😄